active_model_serializers.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.1'
 
-  rails_versions = ['>= 4.1', '< 7.0']
+  rails_versions = ['>= 4.1', '< 8.0']
```

Comment:
as the rails version on `main` branch is now having `7.1.0`, can we use `< 7.1` instead of `< 8.0`. thanks

Reply:
Good idea